### ISSUE TITLE
inspec-core fails when aws gem is installed

### ISSUE THREAD

**Issue by jquick** [inspec/inspec#3257]
### 🎛 Description

**Choose one:** is this a 🐛 bug report 
Issue Description
Jira issue title: Inspec-core fails to require resource_support/aws when aws-sdk gem is installed

Issue Summary: inspec-core will attempt to require resource_support/aws if it detects that an aws-sdk gem is installed. This of course fails because inspec-core does not include any of the aws resource files.
ERROR: Report handler Chef::Handler::AuditReport raised #<LoadError: cannot load such file – resource_support/aws>
In this case inspec-core is getting installed using the Audit 7.0.0 cookbook and the Inspec version is set to 2.2.16. The aws-sdk gem is installed as part of another cookbook and is using v3 which is not compatible with the full inspec gem.
I've also created a Github issue for inspec for this: https://github.com/inspec/inspec/issues/3161
Expected Results: Audit cookbook should run the specified profiles.
Instead of testing for just the presence of the awk-sdk gem it should also test if this is running from inspec or inspec-core to determine if those files should be required.
Actual Results: Audit cookbook fails with exception unable to load resource_support/aws files.

Repro Steps (if possible):

Using the audit 7.0.0 cookbookk Install inspec-core specifying version 2.2.16
Install aws-sdk gem using chef_gem 'aws-sdk'; require 'aws-sdk'
Configure the audit cookbook to run any profile.

Request for Developers: Product Fix
```
[2018-06-21T10:11:04+00:00] ERROR: Report handler Chef::Handler::AuditReport rai 
sed #&lt;LoadError: cannot load such file -- resource_support/aws&gt; 
[2018-06-21T10:11:04+00:00] ERROR: C:/opscode/chef/embedded/lib/ruby/2.5.0/rubyg 
ems/core_ext/kernel_require.rb:59:in require' 
[2018-06-21T10:11:04+00:00] ERROR: C:/opscode/chef/embedded/lib/ruby/2.5.0/rubyg 
ems/core_ext/kernel_require.rb:59:inrequire' 
...
```

**Comment by clintoncwolfe** [comment]
This report specifies an issue for inspec 2.2.16.  A fix for this issue was released in [inspec 2.2.27](https://github.com/inspec/inspec/releases/tag/v2.2.27) based on PR #3163 .

Based on that, I'm closing this as a duplicate.  If I've misunderstood, please of course re-open :-)

**Comment by btm** [comment]
Here was the duplicate issue: https://github.com/inspec/inspec/issues/3161
